providers/universal-provider/src/utils/deepLinks.ts

```diff
@@ -9,18 +9,23 @@ export const handleDeepLinks = (client: InstanceType<typeof SignClient>, params:
     // only handle the request if it matches the request and topic
     if (payload.request !== params.request || payload.topic !== params.topic) return;
     client.events.removeListener("session_request_sent", handleSessionRequestSent);
-    deeplinkRedirect();
+    deeplinkRedirect(params.topic, params.id);
```

Comment:
```suggestion
    deeplinkRedirect(params);
```
isnt it cleaner to pass the whole `params` as a single argument? If we need to use something else from the object it would be available without needing to update the fx arguments